pandas/io/tests/test_pickle.py

```diff
@@ -15,7 +15,7 @@
 from pandas.tseries.offsets import Day, MonthEnd
 
 
-class TestPickle():
+class TestPickle(tm.TestCase):
```

Comment:
I think this was a stray commit from a different branch, trying to get pytest running. Will remove. 

> On Jul 14, 2016, at 19:23, Sinhrks <notifications@github.com> wrote:
> 
> In pandas/io/tests/test_pickle.py:
> 
> > @@ -15,7 +15,7 @@
> >  from pandas.tseries.offsets import Day, MonthEnd
> >  
> >  
> > -class TestPickle():
> > +class TestPickle(tm.TestCase):
> Does it run test generators? see docstring:
> 
>     NOTE: TestPickle can't be a subclass of tm.Testcase to use test generator.
>     http://stackoverflow.com/questions/6689537/
>     nose-test-generators-inside-class
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>